tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -52,33 +61,59 @@ def signin(self, username: str, password: str):
         )
 
         if response.status_code == 503:
-            print(f"Server url {self.url} not available.")
+            print(f"Server '{self.url}' not available.")
             self.session.close()
             sys.exit()
 
-        response_json: dict = response.json()
-
-        status: str = response_json['data']['signin']['status']
-        message: str = response_json['data']['signin']['message']
-
-        if status == 'SUCCESS':
-            log.debug(f"Success: Signed into {self.url} as {username}.")
+        response_json: dict = response.json().get('data', {}).get('signin', {})
+        
+        if response_json.get('status') == 'SUCCESS':
+            message = f"Success: Signed into {self.url} as {username}." 
+            log.info(message)
+            print(message)
             self.signin_status = 'success'
-        elif status == 'FAILED':
+        elif response_json.get('status') == 'FAILED':
+            message = f"Error: Unable to sign into {self.url} as {username}.\n{response_json.get('message')}"
             log.error(message)
+            print(message)
             self.signin_status = 'failed'
         else:
-            log.error('Error: sign in failed, exiting.')
+            log.error(f"Error: sign in failed, exiting. Sign in status: {response_json.get('status')}")
+            print(f"Unable to sign into {self.url} as {username}")
             self.signin_status = 'failed'
 
-    def list_databases(self):
+        self.username = username
+        self.schemas = self.list_schemas()
+
+    def sign_out(self):
+        """Signs the client out of the EMX2 server."""
+        response = self.session.post(
+            url=self.api_graphql,
+            json={'query': queries.sign_out()}
+        )        
+        
+        status = response.json().get('data', {}).get('signout', {}).get('status')
+        message = response.json().get('data', {}).get('signout', {}).get('message')
+        if status == 'SUCCESS':
+            print(f"Signed out of {self.url}")
+        else:
+            print(f"Unable to sign out of {self.url}.")
+            print(message)
+            
+    def status(self):
+        """View client information"""
+        schemas = '\n\t'.join(self.schemas)
+        message = (
+          f"Host: {self.url}\n"
+          f"User: {self.username}\n"
+          f"Status: {'Signed in' if self.signin_status == 'success' else 'Logged out'}\n"
+          f"Schemas: \n\t{schemas}\n"
+        )
+        print(message)
+
+    def list_schemas(self):
         """List the databases present on the server."""
```

Comment:
sorry for creating this confusion on databases vs schemas. I recommend we standardize our terminology: do we have 'schemas' or 'databases'?

If nobody cares I recommend schemas because that is what postgresql calls them.

Reply:
Agreed. The terminology should be consistent.